x-pack/plugins/triggers_actions_ui/public/application/lib/action_variables.test.ts

```diff
@@ -44,10 +44,18 @@ describe('transformActionVariables', () => {
           "description": "The alert action group that was used to scheduled actions for the alert.",
           "name": "alertActionGroup",
         },
+        Object {
```

Comment:
I double-checked the variables we're making available, `alertActionSubgroup` should have been here, but was not, so added it.